test/sycl_tests.cc

```diff
@@ -171,4 +171,46 @@ TEST_CASE_METHOD(test_utils::device_queue_fixture, "SYCL has working simple scal
 
 #endif
 
+TEST_CASE("SYCL implements by-value equality-comparison of device information", "[sycl][device-selection][!mayfail]") {
+	constexpr static auto get_devices = [] {
```

Comment:
Is there a reason why this is `static`? (Same with the other lambda below)

Reply:
Delegated to @fknorr.